burr/core/state.py

```diff
@@ -196,22 +235,31 @@ def get_all(self) -> Dict[str, Any]:
     def serialize(self, **kwargs) -> dict:
         """Converts the state to a JSON serializable object"""
         _dict = self.get_all()
-        return {
-            # TODO: handle field specific custom serialization
-            k: serde.serialize(v, **kwargs)
-            for k, v in _dict.items()
-        }
+
+        def _choose_serde(k, v, **extrakwargs) -> Union[dict, str]:
```

Comment:
Better name than `_choose_serde`? It does more than choosing (I think)

Reply:
sure.